duckduckgo/iOS
# fix about layout

Task/Issue URL: https://app.asana.com/0/276630244458377/892362367975739
Tech Design URL:
CC:

**Description**:

Fixes about scrolling on iPad and fixes tapping the about link in some cases where it wouldn't work.

**Steps to test this PR**:

1. Check the about screen on an iPad, iPhone and small iPhone in both portrait and landscape. It should scroll correctly and tapping the link at the bottom should work.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
